Source/Scene/SkyAtmosphere.js

```diff
@@ -80,6 +80,31 @@ define([
         });
         this._spSkyFromSpace = undefined;
         this._spSkyFromAtmosphere = undefined;
+        
+        // hue, saturation, and brightness shift values for color adjustment
+        /**
```

Comment:
For the doc for hue, saturation, and brightness, can you provide a bit more info like the range of each value an an example value to get a certain effect?  These parameters may not be obvious to all our users.